frontend/app/routes/protected/person-case/parent-details.tsx

```diff
@@ -232,10 +240,11 @@ type FormData = {
 
 interface ParentInformationProps {
   defaultFormValues: FormData[];
+  loaderData: Route.ComponentProps['loaderData'];
   errors?: Record<string, [string, ...string[]] | undefined>;
 }
 
-function ParentInformation({ defaultFormValues, errors }: ParentInformationProps) {
+function ParentInformation({ defaultFormValues, loaderData, errors }: ParentInformationProps) {
```

Comment:
Since `defaultFormValues` is included in `loaderData`, does it make sense to just use the latter here?

Reply:
Yeah.. but it's not really that bad because all the components are local. There are spots where the child component updates the data that belongs to the parent, though (moving items from `defaultFormValues`), which I really dislike.